Scroller/tests/Scroller-specs.js

```diff
@@ -19,167 +20,161 @@ describe('Scroller', () => {
 	});
 
 	describe('Scrollbar visibility', () => {
-		test(
-			'should render both horizontal and vertical scrollbars when \'horizontalScrollbar\' and \'verticalScrollbar\' are "visible"',
-			() => {
-				const subject = mount(
-					<Scroller
-						horizontalScrollbar="visible"
-						verticalScrollbar="visible"
-					>
-						{contents}
-					</Scroller>
-				);
-
-				const expected = 2;
-				const actual = subject.find('Scrollbar').length;
-
-				expect(actual).toBe(expected);
-			}
-		);
+		test('should render both horizontal and vertical scrollbars when \'horizontalScrollbar\' and \'verticalScrollbar\' are "visible"', () => {
+			render(
+				<Scroller
+					horizontalScrollbar="visible"
+					verticalScrollbar="visible"
+				>
+					{contents}
+				</Scroller>
+			);
 
-		test(
-			'should render only vertical scrollbar when \'verticalScrollbar\' is "visible" and \'horizontalScrollbar\' is "hidden"',
-			() => {
-				const subject = mount(
-					<Scroller
-						horizontalScrollbar="hidden"
-						verticalScrollbar="visible"
-					>
-						{contents}
-					</Scroller>
-				);
-
-				const expected = 1;
-				const actual = subject.find('Scrollbar').length;
-
-				expect(actual).toBe(expected);
-			}
-		);
+			const upScrollbarButton = screen.getByLabelText('scroll up');
+			const downScrollbarButton = screen.getByLabelText('scroll down');
+			const leftScrollbarButton = screen.getByLabelText('scroll left');
+			const rightScrollbarButton = screen.getByLabelText('scroll right');
 
-		test(
-			'should not render any scrollbar when when \'horizontalScrollbar\' and \'verticalScrollbar\' are "hidden"',
-			() => {
-				const subject = mount(
-					<Scroller
-						horizontalScrollbar="hidden"
-						verticalScrollbar="hidden"
-					>
-						{contents}
-					</Scroller>
-				);
-
-				const expected = 0;
-				const actual = subject.find('Scrollbar').length;
-
-				expect(actual).toBe(expected);
-			}
-		);
+			expect(upScrollbarButton && downScrollbarButton).toBeInTheDocument();
+			expect(leftScrollbarButton && rightScrollbarButton).toBeInTheDocument();
+		});
+
+		test('should render only vertical scrollbar when \'verticalScrollbar\' is "visible" and \'horizontalScrollbar\' is "hidden"', () => {
```

Comment:
You can leave the description on a separate line as it is too long. (same for the other 2 tests above)